docs/how_to/implement_the_right_toggle_type.rst

```diff
@@ -78,11 +84,11 @@ Unlike the underlying waffle and settings libraries which look up values by name
 
 .. code:: python
 
-    from whereever import FLAG_FOO, SWITCH_BAR, SETTING_TOGGLE_BAZ
+    from whereever import FOO_FLAG, BAR_SWITCH, SETTING_TOGGLE_BAZ
 
-    FLAG_FOO.is_enabled()
-    SWITCH_BAR.is_enabled()
-    SETTING_TOGGLE_BAZ.is_enabled()
+    FOO_FLAG.is_enabled()
```

Comment:
**Note to self:** Add examples with new methods once https://github.com/openedx/edx-toggles/pull/293 lands.